matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/dtos/offers/AbstractMatchedBet.java

```diff
@@ -11,6 +11,7 @@
     protected OddsType oddsType;
     protected Currency currency;
     protected BigDecimal odds;
```

Comment:
i'd say we should replace all bigdecimal for double

Reply:
Matchbook uses `BigDecimal` and it seems reasonable to me because we want full precision in trading math. Odds and currency-related values are part of these equations, hence, in my opinion, all of them have to be `BigDecimal`.

Reply:
i think we should use double because 11 points after dot should be enough for precision. 
BigDecimal size is 32 bytes, where double is 8 bytes, so basically 4 times bigger, for a performance perspective also around 4 times slower.